tests/attacks/test_projected_gradient_descent.py

```diff
@@ -134,14 +137,8 @@ def _test_backend_mnist(self, classifier, x_train, y_train, x_test, y_test):
         # Check that x_test has not been modified by attack and classifier
         self.assertAlmostEqual(float(np.max(np.abs(x_test_original - x_test))), 0.0, delta=0.00001)
 
-        # Test the masking:
-        attack = ProjectedGradientDescent(classifier, num_random_init=1)
-        mask = np.random.binomial(n=1, p=0.5, size=np.prod(x_test.shape))
-        mask = mask.reshape(x_test.shape)
-
-        x_test_adv = attack.generate(x_test, mask=mask)
-        mask_diff = (1 - mask) * (x_test_adv - x_test)
-        self.assertAlmostEqual(float(np.max(np.abs(mask_diff))), 0.0, delta=0.00001)
```

Comment:
Why are the tests for masking removed?

Reply:
This is accidentially removed, it is now recovered.